doc/src/suite-design-guide/style-guide.tex

```diff
@@ -116,8 +116,14 @@ \subsubsection{Script String Lines}
             fi"""
 \end{lstlisting}
 
-But if you do this, watch your line length (see~\ref{Line Length}) and {\em do not
-indent here documents}:
+On parsing the triple quoted value, Cylc will remove any common leading
+whitespace from every lines using the logic of
+\href{https://docs.python.org/2/library/textwrap.html#textwrap.dedent}{Python's textwrap.dedent}
+so the script block would end up being the same as the previous example.
+However, you should watch your line length (see~\ref{Line Length}) when you
+have many levels of indentations.
+
+{\em Do not indent here documents}:
```

Comment:
This is not true on this branch - you just have to be careful to indent here documents to *exactly* the right level (particularly the __EOF__ marker) so that it ends up against the left margin in the job script.